source/magnifier/__init__.py

```diff
@@ -0,0 +1,58 @@
+# A part of NonVisual Desktop Access (NVDA)
```

Comment:
I know this might be annoying, but can you please change this submodule's name from `magnifier` to `_magnifier`. We do this when adding new modules so they remain part of the private API. this indicates to add-on authors that the code isn't stable yet, which allows us to refactor the code without breaking add-ons. I suggest using a refactor tool for whatever IDE you are using. [VS code instructions](https://github.com/microsoft/vscode-docs/blob/vnext/release-notes/v1_63.md#python)